content/index.mdx

```diff
@@ -47,8 +47,12 @@ Here are some case studies:
 
 * [John Lindquist: The Original egghead 🥚](/instructor/case-studies/john-lindquist)
 * [Kent C. Dodds, egghead instructor](/instructor/case-studies/kent-c-dodds)
+* [Laurie Barth, egghead instructor](/instructor/case-studies/laurie-barth)
+* [Eve Porcello, egghead instructor](/instructor/case-studies/eve-porcello)
```

Comment:
I followed what was already there labeling the rest as `egghead instructor` but the repetitiveness feels off